src/main/pages/setup-docker/docker-cli.adoc

```diff
@@ -282,47 +355,49 @@ The Che CLI is constructed of multiple Docker images within the Che source repos
 /dockerfiles/init  # Manifests used to configure Che on a host installation
 ----
 
-The Che CLI is authored in `bash`. The `cli` image depends upon both the `base` image and the `init` image. In the source repository, we have `build.sh` commands which will build these Docker images for you either one at a time or collectively as a group.
+The Che CLI is authored in the `bash` script. The `cli` image depends upon both the `base` image and the `init` image. In the source repository, the `build.sh` commands builds these Docker images either one at a time or collectively as a group.
```

Comment:
Typo: "the `build.sh` commands build"

Also, what does "the `bash` script" mean here? The original wording made sense to me, but I'd change `bash` to Bash:

"The Che CLI is written in Bash."